--- x264-snapshot-20170730-2245-stable/configure.0001~	2017-07-31 09:46:26.818914012 +0200
+++ x264-snapshot-20170730-2245-stable/configure	2017-07-31 09:50:07.175652142 +0200
@@ -891,6 +891,7 @@ fi
 if [ $asm = auto -a $ARCH = ARM ] ; then
     # set flags so neon is built by default
     [ $compiler == CL ] || echo $CFLAGS | grep -Eq '(-mcpu|-march|-mfpu)' || CFLAGS="$CFLAGS -mcpu=cortex-a8 -mfpu=neon"
+    CFLAGS="$CFLAGS -Wa,-march=armv7-a -Wa,-mfpu=neon"
 
     cc_check '' '' '__asm__("add r0, r1, r2");' && define HAVE_ARM_INLINE_ASM
     if [ $compiler = CL ] && cpp_check '' '' 'defined(_M_ARM) && _M_ARM >= 7' ; then
