diff -up x264-stable-5db6aa6cab1b146e07b60cc1736a01f21da01154/configure.1~ x264-stable-5db6aa6cab1b146e07b60cc1736a01f21da01154/configure
--- x264-stable-5db6aa6cab1b146e07b60cc1736a01f21da01154/configure.1~	2021-09-01 01:30:53.783596575 +0200
+++ x264-stable-5db6aa6cab1b146e07b60cc1736a01f21da01154/configure	2021-09-01 01:31:45.344886772 +0200
@@ -1266,7 +1266,7 @@ if [ "$gpac" = "auto" -a "$lsmash" != "y
         fi
     fi
     if [ -z "$GPAC_LIBS" ] ; then
-        GPAC_LIBS="-lgpac_static"
+        GPAC_LIBS="-lgpac"
         cc_check "" -lz && GPAC_LIBS="$GPAC_LIBS -lz"
         cc_check "" -ldl && GPAC_LIBS="$GPAC_LIBS -ldl"
         if [ "$SYS" = "WINDOWS" ] ; then
